src/executables/util_cs.c

```diff
@@ -3342,7 +3364,9 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
     check_master_info_end:
       if (error != NO_ERROR)
 	{
-	  PRINT_AND_LOG_ERR_MSG ("%s\n", db_error_string (3));
+	  PRINT_AND_LOG_ERR_MSG ("\nERROR : %s\n", db_error_string (3));
+	  check_master_info = false;
```

Comment:
It makes `if (check_copied_info && check_master_info)` to be false when `check_copied_info` it true. `*** Delay in Copying Active Log ***` will be skipped below. Is it intended?

Reply:
Right. It is intended. 
I think `*** Delay in Copying Active Log ***` should be printed when correct log files are copied from correct host